src/main/java/uk/gov/pay/connector/gateway/stripe/StripeSdkClient.java

```diff
@@ -48,13 +49,16 @@ public void deleteCustomer(String customerId, boolean live) throws StripeExcepti
     private String getStripeApiKey(boolean live) {
         return live ? stripeGatewayConfig.getAuthTokens().getLive() : stripeGatewayConfig.getAuthTokens().getTest();
     }
-
+    
     public Refund getRefund(String stripeRefundId, boolean live) throws StripeException {
         String apiKey = getStripeApiKey(live);
         RequestOptions requestOptions = RequestOptions.builder()
                 .setApiKey(apiKey)
                 .build();
+        var expandList = List.of("charge");
+        Map<String, Object> params = new HashMap<>();
+        params.put("expand", expandList);
```

Comment:
You could make this a little simpler:

```suggestion
        Map<String, Object> params = Map.of(
                "expand", List.of("charge"));
```

Reply:
Thanks, I will address it in the next PR.